### PR TITLE
add keyframes-animation to nav and some files to img-folder

### PR DESCRIPTION
**Pull request by martinhaiss**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,8 +50,8 @@ <h2>Fähigkeiten</h2>
   <section id="contact">
     <h2>Kontakt</h2>  
     <p>Martin Haiß</p>
-    <p>tel 0157.58831813</p>
-    <a href="mailto:haissmartin@gmail.com">Email me</a>
+    <p>phone: 0157.58831813</p>
```
Comment (macfrei) [inline index.html:53]: You could check out how to display phone numbers in HTML and also the `address` element.

File: styles.css [hunk 2]
```diff
@@ -19,11 +19,33 @@ html {
 
  a {
   color: white;
+  /*font-size: 100%;*/
```
Comment (macfrei) [inline styles.css:22]: Commented out code should be deleted before merging into the `main` branch 🙂

File: styles.css [hunk 3]
```diff
@@ -19,11 +19,33 @@ html {
...
+  animation-name: color;
+  animation-duration: 700ms;
+  animation-fill-mode: forwards;
+}
+
+@keyframes color {
+  100% {
+    color: red;
+    }
+}
```
Comment (macfrei) [inline styles.css:42]: Really cool that you tried this! 👍

File: styles.css (hunk 2 above)
Comment (martinhaiss) [reply styles.css:22]: Ah, good to know. Thx! 🤓

File: index.html (hunk 1 above)
Comment (martinhaiss) [reply index.html:53]: Nice, thx!

**Pull request merged**